sample_data_config/crossref-event/crossref-event-data-pipeline.config.yaml

```diff
@@ -18,5 +18,5 @@ schemaFile:
 # the bigquery data warehouse
 publisherIdPrefixes:
   - '10.7554'
-crossrefEventBaseUrl: 'https://api.eventdata.crossref.org/v1/events?rows=100&mailto=h.ciplak@elifesciences.org'
+CrossrefEventBaseUrl: 'https://api.eventdata.crossref.org/v1/events?rows=100&mailto=h.ciplak@elifesciences.org'
```

Comment:
just a minor comment: starting with a lower case letter would probably be more consistent with the other properties?